backend/services/test.py

```
@@ -0,0 +1,188 @@
+            "Authorization": f"Bearer {self.api_key}",
+            "Accept": "application/json"
+        }
+        matches = []
+        page = 1
+        
+        while True:
+            paginated_url = f"{url}{page}"
+            self.logging.info(f"Fetching page {page} from URL: {paginated_url}")
+            
+            try:
+                response = requests.get(paginated_url, headers=headers)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)